app/lib/features/home/widgets/sidebar_widget.dart

```diff
@@ -26,100 +26,123 @@ class SidebarWidget extends ConsumerWidget {
     final sidebarNavItems = ref.watch(sidebarItemsProvider(context));
     final isGuest = ref.watch(alwaysClientProvider).isGuest();
 
-    return AdaptiveScaffold.standardNavigationRail(
-      // main logic
-      destinations: sidebarNavItems,
-      selectedIndex: navigationShell.currentIndex,
-      onDestinationSelected: (tabIndex) {
-        if (sidebarNavItems[tabIndex].location != null) {
-          final item = sidebarNavItems[tabIndex];
-          // go to the initial location of the selected tab (by index)
-          if (item.isSpaceTab) {
-            context.go(item.location!);
-          } else if (item.pushToNavigate) {
-            context.push(item.location!);
-          } else {
-            navigationShell.goBranch(
-              tabIndex,
-              initialLocation: tabIndex == navigationShell.currentIndex,
-            );
-          }
-        }
-      },
-
-      // configuration
-      labelType: labelType,
-      selectedIconTheme: const IconThemeData(size: 18, color: Colors.white),
-      unselectedIconTheme: const IconThemeData(size: 18, color: Colors.white),
-      padding: const EdgeInsets.all(0),
-      leading: Column(
-        mainAxisAlignment: MainAxisAlignment.center,
-        children: [
-          Visibility(
-            visible: !isGuest,
-            child: Container(
-              key: Keys.avatar,
-              margin: const EdgeInsets.only(top: 8),
-              child: InkWell(
-                onTap: () => context.goNamed(Routes.settings.name),
-                child: const UserAvatarWidget(size: 20),
-              ),
-            ),
-          ),
-          Container(
-            height: 1,
-            margin: const EdgeInsets.symmetric(horizontal: 18, vertical: 10),
-            color: Theme.of(context).colorScheme.primaryContainer,
+    return Stack(
+      children: [
+        Container(
+          margin: const EdgeInsets.only(top: 80),
+          width: 70,
+          child: Column(
+            children: sidebarNavItems
+                .map(
+                  (sidebarNav) => Container(
+                    key: sidebarNav.tutorialGlobalKey,
+                    margin: const EdgeInsets.all(12.0),
+                    height: 40,
+                    width: 40,
+                  ),
```

Comment:
rendering them twice feels a bit like a hack, but whatever...

Reply:
Not getting you by rendering it twice, for info dialog position management there were two `containers`. The outer one is for specifying desktop navbar width area and inter one is for specifying individual tab area. still let me know if there is any scope of improvement here.